packages/cloud-cognitive/src/components/Datagrid/styles/addons/_CustomizeColumnsModal.scss

```diff
@@ -13,30 +13,45 @@
     position: inherit;
     display: flex;
     flex-flow: column;
-    /* stylelint-disable-next-line declaration-no-important */
-    padding-top: 0 !important;
+    padding: 0;
+    margin: 0;
   }
 }
 
-.#{$block-class}__customize-columns-checkbox {
+.bx--form-item.bx--checkbox-wrapper.#{$block-class}__customize-columns-checkbox-wrapper {
```

Comment:
Can you add the carbon prefix variable?
```scss
.#{$block-class} .#{$carbon-prefix}--form-item.#{$carbon-prefix}--checkbox-wrapper.#{$block-class}__customize-columns-checkbox-wrapper
```

Reply:
Removed the `form-item` and `checkbox-wrapper` selectors as they are unecessary